src/systemTest/java/com/flightstats/hub/system/service/ChannelItemPathPartsBuilder.java

```diff
@@ -0,0 +1,42 @@
+package com.flightstats.hub.system.service;
+
+import com.flightstats.hub.model.ChannelItemPathParts;
+import com.google.inject.Inject;
+import org.joda.time.DateTime;
+
+public class ChannelItemPathPartsBuilder {
```

Comment:
There's this and then ChannelItemPathParts also has ```@Builder```, which is a bit confusing. Maybe override Lombok's default implementation of ```@Builder``` with this? (you can override any part of what Lombok creates and it'll create the bits you haven't overridden, see the docs on Builder https://projectlombok.org/features/Builder).

There are multiple classes related to ChannelItemPathParts fulfilling tightly related responsibilities. It feels like they want to live together in one class.